---
output:
  html_document: default
  pdf_document: default
---
The syllabus (mostly the schedule) will be updated throughout the semester. You can see it as a [PDF](https://github.com/bomeara/phylometh/raw/master/other/Syllabus.pdf) or [RMarkdown](https://github.com/bomeara/phylometh/blob/master/other/Syllabus.Rmd) file; I have also embedded it below.


## Phylogenetic Methods (PhyloMeth). Spring 2017

UT students may sign up at EEB 603, CRN 30253 (grad students) or EEB 504, CRN 31819 (advanced undergrads)

Syllabus version: `r format(Sys.time(), '%I:%M %p, %d %B, %Y')`

Instructor: **[Brian O'Meara](http://www.brianomeara.info)** (bomeara@utk.edu)

Course website: http://phylometh.org

Office Hours (446 Hesler) by appointment.

####Anonymous feedback
http://www.brianomeara.info/feedback. This is the first time the course is being run, and it's taught in a very different way from most courses. There is a lot that will go wrong. Let me know what's not working while it can still improve.

#### What you will learn
This is a modern class in phylogenetic methods. You will learn how to get data, build a tree, use comparative methods, and how to make new methods. More importantly, you will learn _why_ to do all this.

#### Requirements
* A laptop on which you can install software (have root / admin access)
    + It'll be easiest if this is a Mac; less easy is Linux; Windows will be hard
* A question and some data to answer it
* GitHub account
* Gitter
* R
* And other software we'll learn about during the course

I am writing a textbook as part of the course
* [HTML version](https://bookdown.org/bomeara/comparative-methods/)
* [PDF](https://bookdown.org/bomeara/comparative-methods/comparative-methods.pdf)

It will be updated as we go, so check back frequently.

#### Evaluation
Grading will be based on effort and performance. Some of the things will be harder for some students than others just given differences in what they've been exposed to so far, and I don't want to penalize students who are less skilled at R, for example. But I do require that you put in work on this class: work through the exercises, think about the papers, etc. Much of the homework is chewy, so it will take some thought to get through, and you might not be able to do all of this. That's ok.

* EEB603 final project will be done solo (though consulting is ok)
* EEB504 final project will be done in pairs

#### Teaching
The class is flipped: rather than me droning at the front of the room, I'll be releasing videos online for you to watch. They will be linked to from this syllabus. In class, talk about what was not clear from the videos about methods, and we'll dig into that. We can also work on problematic parts of the exercises (you should do most of them before class) and talk about papers.

#### Online option
There is an online option, as well. In this second running of the class, I'm not seeking to give out credit, but online visitors are encouraged to sign up.

### Schedule

* Week 1, Jan 12
    + Topic: Getting ready
    + Items: R, github, gitter, unit testing
    + Instructions: http://rpubs.com/bomeara/phylomethsoftwareinstall
    + Exercise: [Getting Started](https://github.com/bomeara/phylometh/blob/master/exercises/gettingstarted/GettingStarted.Rmd)
    + One way to do the exercise is to fork the [github repo](https://github.com/bomeara/phylometh), clone it to your computer, start editing documents (such as those in `exercises/gettingstarted`), then `git commit -m"done homework" -a`, then do a pull request.

* Week 2, Jan 19
    + Topic: Gathering data
    + Reading: [Chapter 3](https://bookdown.org/bomeara/comparative-methods/getting-data-and-trees-into-r.html) of the textbook.
    + Videos: See videos in this [playlist](https://www.youtube.com/playlist?list=PLrJfyqNDuTkMx_1OV_97S0jgdfmHfQ48x)
    + Items: Sequences, trees, name resolution, alignment
    + Exercise: Getting trees: https://github.com/bomeara/phylometh/blob/master/exercises/gettingtreesanddata/exercise.R
    + In class: Discussion of what kinds of data you need for your questions, what kind of trees you need.

* Week 3, Jan 26
    + Topic: Building trees
    + Videos: [Terminology](https://www.youtube.com/watch?v=XSp1gEKNuIo), [Tree space](https://www.youtube.com/watch?v=SQw9ZWFp178), [Likelihood vs Bayes](https://youtu.be/xOADfntkBrs)
    + Reading
        + [Felsenstein 1978](https://www.mendeley.com/catalog/cases-parsimony-compatibility-methods-positively-misleading/)
        + [Felsenstein 1985](https://www.mendeley.com/catalog/confidence-limits-phylogenies-approach-using-bootstrap-55/)
        + [Lewis 2001](https://www.mendeley.com/catalog/phylogenetic-systematics-turns-new-leaf/)
        + [Drummond & Rambaut 2007](https://www.mendeley.com/catalog/beast-bayesian-evolutionary-analysis-sampling-trees-1/)
    + Items: Likelihood; Bayes; Parsimony
    + Potential exercises (choose your own)
        + [RAxML tutorial](http://sco.h-its.org/exelixis/web/software/raxml/hands_on.html): Fast maximum likelihood tree inference program.
        + [RevBayes tutorials](https://revbayes.github.io/tutorials.html): Bayesian program with many different models.
        + [BEAST tutorials](http://treethinkers.org/tutorials/divergence-time-estimation-using-beast/): Bayesian tree inference program, especially used for getting chronograms.
        + [Cyverse discovery environment](https://de.cyverse.org/de/): Free service (thanks, NSF!) for running phylogenetic (and other) software
        

* Week 4, Feb 2
    + Topic: Building trees
    + Items: Gene tree species tree; Network
    + Reading
        + [Maddison 1997](https://doi.org/10.1093/sysbio/46.3.523): Gene trees in species trees
        + [Heled & Drummond 2009](https://doi.org/10.1093/molbev/msp274): A method for inferring species trees given gene trees inside them
        + [Degnan & Rosenberg 2000](http://dx.doi.org/10.1016/j.tree.2009.01.009): Cases where the most frequent gene tree does not match the species tree
      


* Week 5, Feb 9
    + Topic: Dating trees
    + Items: Beast, r8s


* Week 6, Feb 16
    + Topic: Correlating characters
    + Items: independent contrasts, Pagel 1994, Hansen OU


* Week 7, Feb 23
    + Topic: Continuous character models
    + Items: Surface, OUwie, geiger


* Week 8, Mar 2
    + Topic: Discrete character models
    + Items: Pagel 1994, corHMM, geiger


* Week 9, Mar 9
    + Topic: Guest lecture


* Week 10, Mar 23
    + Topic: SSE models
    + Items: diversitree, hisse


* Week 11, Mar 30
    + Topic: Simulating data
    + Items: TreeSim, geiger
    
* Week 12, Apr 6
    + Topic: Building a new comparative method

* Week 13, Apr 13
    + Topic: Testing a new comparative method

* Week 14, Apr 20
    + Topic: Student choice

* Week 11, April 27
    + Topic: Student choice

**Course made possible by**:  NSF CAREER grant to O'Meara & ongoing support from the Department of Ecology & Evolutionary Biology, U. of Tennessee, Knoxville.